js/src/dapps/localtx/Transaction/transaction.js

```diff
@@ -256,23 +259,29 @@ export class LocalTransaction extends BaseTransaction {
 
     const newTransaction = {
       from: transaction.from,
-      to: transaction.to,
```

Comment:
Ah, just noticed that we're adding it later on. I don't understand how does it fix the problem though. Previously `transaction.to` was set to `0x` so we will set it anyway, no?